config/module.ini

```diff
@@ -1,9 +1,9 @@
 [info]
 version = "1.4.0"
 omeka_version_constraint = "^3.0.0"
-name = "File Sideload"
-description = "Add files that are already on your server to items."
-author = "Omeka Team"
+name = "File and Directory Sideload"
```

Comment:
We're not going to change the name of the module, but we're happy to have you attributed as an author.

Reply:
fixed